index.html

```diff
@@ -220,7 +235,7 @@ <h2 class="footer_title">Contact me</h2>
                 <input type="text" id="name" name="name" class="input" placeholder="Your Name" maxlength="30" required>
                 <input type="email" id="email" name="email" class="input" placeholder="Your Email" required>
                 <textarea name="message" id="message" cols="30" rows="10" class="textarea input" placeholder="Write your message here..." required maxlength="500"></textarea>
-                <button class="footer_button button" type="submit">Get in touch</button>
+                <button class="footer_button button" type="submit"><b>Get in touch</b></button>
```

Comment:
Please consider using the same styles for the button as used in the Figma Template. Currently, your button is bigger in width and is aligned to the left side ↩️ whether in the Design template, it's in the center.

## YOUR DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/228924647-e0dd4dd3-733a-4c61-a205-08ab6f3c58f3.png)


## FIGMA DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/228924880-91c676f5-94a7-453e-a3a4-1be96f64f086.png)